FSharpWorkspaceShim/FSharpWorkspaceShim.fsproj

```diff
@@ -18,7 +18,11 @@
 
   <ItemGroup>
     <PackageReference Include="FSharp.Compiler.Service" />
-    <PackageReference Include="Microsoft.CodeAnalysis" Version="3.6.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis" Version="3.11.0" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Update="FSharp.Core" Version="5.0.2" />
```

Comment:
Don't specify F# package versions; they're all handled in the root `Directory.Build.targets`.